integrations/cdi/datasource/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>jakarta.annotation-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Not sure that belongs in this PR.

Reply:
it was required to build since some of the scopes changed from provided to runtime as part of this change.